recipes/mocket/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set python_min = "3.8" %}
+{% set name = "mocket" %}
+{% set version = "3.13.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/mocket-{{ version }}.tar.gz
+  sha256: 1a6b3658e668c2bc1fe8442df7840b5b77318544f5c157a9952776b17ebff2a2
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - hatchling >=1.22.2
+    - pip
+  run:
+    - python >={{ python_min }}
+    - puremagic
+    - decorator >=4.0.0
+    - urllib3 >=1.25.3
+    - h11
+
+test:
+  imports:
+    - mocket
+  commands:
+    - pip check
+  requires:
+    - python {{ python_min }}
+    - pip
+
+about:
+  home: https://github.com/mindflayer/python-mocket/mocket
```

Comment:
This link is broken. Should probably be https://github.com/mindflayer/python-mocket or https://github.com/mindflayer/python-mocket/tree/main/mocket